software/firmware/bootloader.py

```diff
@@ -144,4 +159,11 @@ def main(self):
             europi.b1._handler_both(europi.b2, self.exit_to_menu)
             europi.b2._handler_both(europi.b1, self.exit_to_menu)
 
-            script_class().main()
+            try:
+                script_class().main()
+            except Exception:
```

Comment:
```suggestion
            except Exception as err:
```

Reply:
[required] The exception handling code itself crashes without this change.

Reply:
Whoops. I had that saved and just forgot to commit it. Sorry about that.